methcomp/regression.py

```diff
@@ -1,532 +1,279 @@
     title : str, optional
-        Title of the linear regression plot. If None is provided, no title will be plotted.
-    CI : float, optional
-        The confidence interval employed in the linear regression line. Defaults to 0.95.
+        Title of the regression plot. If None is provided, no title will be plotted.
     line_reference : bool, optional
-        If True, a grey reference line at y=x will be plotted in the plot.
-        Defaults to true.
+        If True, a grey reference line at y=x will be plotted in the plot
+        (default: True)
     line_CI : bool, optional
         If True, dashed lines will be plotted at the boundaries of the confidence intervals.
-        Defaults to false.
+        (default: False)
     legend : bool, optional
-        If True, will provide a legend containing the computed Linear regression equation.
-        Defaults to true.
-    color_points : str, optional
-        Color of the individual differences that will be plotted.
-        Color should be provided in format compatible with matplotlib.
-    color_paba : str, optional
-        Color of the mean difference line that will be plotted.
-        Color should be provided in format compatible with matplotlib.
+        If True, will provide a legend containing the computed regression equation.
+        (default: True)
     square : bool, optional
         If True, set the Axes aspect to "equal" so each cell will be
-        square-shaped.
-    point_kws : dict of key, value mappings, optional
-        Additional keyword arguments for `plt.scatter`.
-    ax : matplotlib Axes, optional
-        Axes in which to draw the plot, otherwise use the currently-active
-        Axes.
+        square-shaped. (default: True)
+    ax : matplotlib.axes.Axes, optional
+        matplotlib axis object, if not passed, uses gca()
+    point_kws : Optional[Dict], optional
+        Additional keywords to plt
+    color_regr : Optional[str], optional
+        color for regression line and CI area
+    alpha_regr : Optional[float], optional
+        alpha for regression CI area
 
     Returns
     -------
-    ax : matplotlib Axes
-        Axes object with the linear regression plot.
+    matplotlib.axes.Axes
+        axes object with the linear regression plot
 
     See Also
     -------
-    ..............
+    regressor.Linear - class that implements linear regression and plot
     """
-
-    plotter: _Linear = _Linear(method1, method2,
-                               x_label, y_label, title,
-                               CI, line_reference, line_CI, legend,
-                               color_points, color_regr,
-                               point_kws)
-
-    # Draw the plot and return the Axes
-    if ax is None:
-        ax = plt.gca()
-
-    if square:
-        ax.set_aspect('equal')
-
-    plotter.plot(ax)
-
-    return ax
+    return Linear(method1=method1, method2=method2, CI=CI).plot(
```

Comment:
see earlier comment about returning class vs class.plot

Reply:
Awaiting answer for above :)